docs/development/submitting-patches.rst

```diff
@@ -27,28 +25,38 @@ You can check if your code meets our automated requirements by running
 
 `Write comments as complete sentences.`_
 
-Every code file must start with the boilerplate notice of the Apache License.
-Additionally, every Python code file must contain
+Every Python code file must contain:
 
 .. code-block:: python
 
-    from __future__ import absolute_import, division, print_function
+    from __future__ import absolute_import, division
 
 
 Tests
 -----
 
-All code changes must be accompanied by unit tests with 100% code coverage (as
-measured by the tool `coverage`_.  To test coverage, use our ``tox`` job:
+All code changes must be accompanied by unit tests with 100% code coverage
+(as measured by the tool `coverage`_.)
+
+To test code coverage you'll need to install `detox`_ and `coverage`_.
+They can be installed by running:
 
 .. code-block:: console
 
-    $ tox -e cover
-    Name                                   Stmts   Miss  Cover   Missing
-    --------------------------------------------------------------------
-    ...
-      cover: commands succeeded
-      congratulations :)
+   pip install --user requirements/toolchain.txt
+
+(Or you may prefer to install those requirements in a ``virtualenv``.)
+
+Then run:
+
+.. code-block:: console
+
+    $ coverage erase \
+      && detox \
+      && coverage combine \
+      && coverage html
+
+And open ``htmlcov/index.html`` in your web browser.
```

Comment:
I took this from @glyph 's comment in https://github.com/rackerlabs/mimic/pull/375#issuecomment-155905468